lightning/src/ln/channelmanager.rs

```diff
@@ -8774,7 +8780,8 @@ where
 			Some(payer_note) => builder.payer_note(payer_note),
 		};
 		let invoice_request = builder.build_and_sign()?;
-		let reply_path = self.create_blinded_path().map_err(|_| Bolt12SemanticError::MissingPaths)?;
+		let reply_paths = self.create_blinded_path(5)
```

Comment:
Could you add a constant for this?

Reply:
Done!
Introduce a new `MAX_REPLY_PATHS` constant and comment it's meaning appropriately [4032289](https://github.com/lightningdevkit/rust-lightning/commit/40322898323c2656779c1ff23e54310229ecd671)
Thanks!